app/scripts/modules/core/loadBalancer/filter/LoadBalancerFilterCtrl.js

```diff
@@ -52,7 +52,7 @@ module.exports = angular.module('spinnaker.core.loadBalancer.filter.controller',
     this.initialize = function() {
       ctrl.accountHeadings = getHeadingsForOption('account');
       ctrl.regionHeadings = getHeadingsForOption('region');
-      ctrl.stackHeadings = getHeadingsForOption('stack');
+      ctrl.stackHeadings = ['(none)'].concat(getHeadingsForOption('stack'));
```

Comment:
I would prefer to see the concat of "none" encapsulated in the the list in the private getHeadingsForOption function.